installer/templates/new/priv/gettext/errors.pot

```diff
@@ -0,0 +1,100 @@
+#################################################################
```

Comment:
Ugh, I just noticed those comments are copied to the `.po` file. let's drop them from here and add it to `web/gettext.ex`.